nevergrad/functions/multiobjective/core.py

```diff
@@ -127,4 +128,4 @@ def pareto_front(self, size: Optional[int] = None, subset: str = "random") -> Li
                             raise ValueError(f'Unknown subset for Pareto-Set subsampling: "{subset}"')
                     score += best_score ** 2
                 scores += [score]
-        return [p[0] for p in possibilities[scores.index(min(scores))]]
+        return [p[0] for p in possibilities[scores.index(min(scores))]], scores
```

Comment:
Do we really need the scores? That makes the API difficult to use. If we do we need another solution